in_toto/gpg/common.py

```diff
@@ -347,12 +400,10 @@ def _get_verified_subkeys(bundle):
   """
   <Purpose>
     Helper function to verify the subkey binding signature for all subkeys in
-    the passed bundle. Only valid (i.e. parsable) subkeys that are verifiably
-    bound to the the master key of the bundle are returned. All other subkeys
-    are discarded.
-
-    TODO: Extract additional information about subkeys from the signatures
-    (see inline TODO below).
+    the passed bundle in order to enrich subkeys with additional information
+    (e.g. expiration dates). Only valid (i.e. parsable) subkeys that are
```

Comment:
Note to self (and everybody) `parsable` is a word (unlike parse-able, which is an ugly composite word)